configure.ac

```diff
@@ -75,6 +75,13 @@ if test "x$with_systemdsystemunitdir" != xno; then
 fi
 AM_CONDITIONAL(HAVE_SYSTEMD, [test -n "$with_systemdsystemunitdir" -a "x$with_systemdsystemunitdir" != xno ])
 
+AC_ARG_WITH(xmlto,
+        [AC_HELP_STRING([--with-xmlto],
+                [Enable xmlto manual generation @<:@default=yes@:>@])],
+        [],
+        [with_xmlto=no])
```

Comment:
@jameshilliard This will cause xmlto isn't going to be used if ```---with-xmlto``` isn't applied, which contradicts with the option's comment. And it would be great if xmlto was used by default and its usage would be skipped only if an user adds ```--without-xmlto```.

Reply:
Oh, must not have tested that correctly, think it should be fixed now.

Reply:
@jameshilliard tbh you could only change ```with_xmlto=no``` to ```with_xmlto=yes``` and it would do the trick, but the current implementation works too.